### ISSUE TITLE
A11y_mssql for VSCode_Message Window_ScreenReader:Screen reader is announcing the "message" dropdown as collapsed when it is in expanded state and vise versa.

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode-mssql#17198]
## Actual:
Screen reader is announcing the "message" dropdown as collapsed when it is in expanded state and vise versa.

**Similar issue is observed with Jaws**
**Similar issue is also observed in Select top 1000 Results window_Results and messages window.**

## Expected:
Screen reader should announce the "message" dropdown as expanded when it is in expanded state and collapsed when it is in collapsed state.

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev2;#AINotInScope;#WCAG1.3.1;#NVDA;#Jaws;#STP;#Element:Dropdown;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11
Screen Reader: NVDA(2021.3), Jaws(2021.2103.174)

## Repro Steps:

1. Open Visual studio code 
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server  button
4. Tab till Add Connection Button and insert the valid credentials
5. Right click on the server/database node you want to run the new query on.
6. Press enter “New Query” write any valid query and click on "Execute query button"
7. Message Window will open
8. Run NVDA and tab to the "message" dropdown and check if the NVDA is announcing "message" dropdown as collapsed when it is in expanded state and vise versa.

## User Impact:
Screen reader users will get confused if the screen reader is announcing all the state of dropdown incorrectly

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## Attachments
![17198](https://user-images.githubusercontent.com/86601127/149266325-4aed8966-506a-4c56-a08e-1236ad1590b7.gif)
![17198 2](https://user-images.githubusercontent.com/86601127/149269000-a475bd7c-e6ba-4350-bdfb-25bd47aa9876.gif)

**Comment by smartguest** [comment]
We need to change the collapsed state of the dropdown in the section of vscode-mssql, making sure it does not break the dropdown when this is done, currently this is done in app.component.ts as an html template.

**Comment by Roopesh-T** [comment]
Above issue is not repro.
[Presentation3.pptx](https://github.com/microsoft/vscode-mssql/files/9082654/Presentation3.pptx)
